docs/minimal.toml

```diff
@@ -1,3 +1,6 @@
+[node]
+addr = "ceresdb0"
```

Comment:
It is weird to use a name for addr.

Reply:
is this a required config, even for standalone mode?